src/App.tsx

```diff
@@ -0,0 +1,61 @@
+import { Component } from 'react';
+import ErrorBoundary from './components/error-boundary/ErrorBoundary';
```

Comment:
в одной строчке трижды ErrorBoundary
я бы сделал так
```suggestion
import ErrorBoundary from './components/error-boundary';
```

Reply:
Спасибо за полезное замечание, учтем в будущем. 
При работе над этим заданием мы не выносили экспорт компонентов в index файл. Вносить изменение для одного ErrorBoundary не целесообразно.